service/models.py

```diff
@@ -134,3 +138,9 @@ def find(cls, inventory_item_id):
         """
         logger.info("Processing lookup for id %s ...", inventory_item_id)
         return cls.query.get(inventory_item_id)
+
+    @classmethod
+    def all(cls):
```

Comment:
I needed to add this to make the tests for `InventoryItem.delete()` meaningful. 

If we can't assert how many items are in the database before creating and after deleting it's pretty much impossible to assert the behavior is correct.